admin_guide/production_sizing.rst

```diff
@@ -0,0 +1,38 @@
+.. _production_sizing:
+
+Production Sizing
+$$$$$$$$$$$$$$$$$
+
+This section explores configuration settings for running a cluster in production.
+
+.. _prod_shard_count:
+
+Shard Count
+===========
+
+The number of nodes in a cluster is easy to change (see :ref:`scaling_out_cluster`), but the number of shards to distribute among those nodes is more difficult to change after cluster creation. Choosing the shard count is a balance between the flexibility of having more shards, and the overhead for query planning and execution across them.
+
+The optimal choice varies depending on your access patterns for the data. For instance, in the :ref:`mt_blurb` use-case we recommend choosing between **32 - 128 shards**.  For smaller workloads say <100GB, you could start with 32 shards and for larger workloads you could choose 64 or 128. This means that you have the leeway to scale from 32 to 128 worker machines.
```

Comment:
The real-time analytics answer makes sense to me, but I don't have a clear understanding for this one. We say "you could start with 32 shards", which implies this can be changed later. So we should change that.
Also, I don't think anyone would not choose 64 or 128 based on this recommendation? We don't really say why you would want to choose 32 or 64 or 128. @saicitus/ @samay-sharma  thoughts?